packages/transport/src/modules/accounts/IdentityDeletionProcessController.ts

```diff
@@ -0,0 +1,90 @@
+import { log } from "@js-soft/ts-utils";
+import { ControllerName, TransportController } from "../../core";
+import { IdentityDeletionProcessStatusChangedEvent } from "../../events";
+import { AccountController } from "../accounts/AccountController";
+import { SynchronizedCollection } from "../sync/SynchronizedCollection";
+import { IdentityDeletionProcessClient } from "./backbone/IdentityDeletionProcessClient";
+import { IdentityDeletionProcess } from "./data/IdentityDeletionProcess";
+import { IdentityDeletionProcessStatus } from "./data/IdentityDeletionProcessStatus";
+
+export class IdentityDeletionProcessController extends TransportController {
+    public identityDeletionProcessClient: IdentityDeletionProcessClient;
+    public identityDeletionProcessCollection: SynchronizedCollection;
+
+    public constructor(parent: AccountController) {
+        super(ControllerName.Identity, parent);
+
+        this.identityDeletionProcessClient = new IdentityDeletionProcessClient(this.config, this.parent.authenticator);
+    }
+
+    @log()
+    public override async init(): Promise<IdentityDeletionProcessController> {
+        await super.init();
+
+        this.identityDeletionProcessCollection = await this.parent.getSynchronizedCollection("IdentityDeletionProcesses");
+        return this;
+    }
+
+    private async updateIdentityDeletionProcess(identityDeletionProcess: IdentityDeletionProcess): Promise<void> {
+        const oldIdentityDeletion = await this.getIdentityDeletionProcess(identityDeletionProcess.id.toString());
+        await this.identityDeletionProcessCollection.update(oldIdentityDeletion, identityDeletionProcess);
+        this.eventBus.publish(new IdentityDeletionProcessStatusChangedEvent(this.parent.identity.address.toString(), identityDeletionProcess));
```

Comment:
I assume that when the underlying Backbone routes are called, it is ensured that a status change has actually taken place? I'm not sure if this is appropriate in this file, but perhaps you could still check here whether the status of `oldIdentityDeletionProcess` is different from the status of the new `identityDeletionProcess`.